portfolio/src/main/webapp/exp.html

```diff
@@ -8,31 +8,31 @@
   </head>
   <body>
 
-    <div id='headerPos' class='pageTop underline'>
+    <div class="page-top underline">
         <h1>Ali's Portfolio</h1>
     </div>
 
-    <div class='navBar'>
+    <div class="navBar">
         <ul>
-            <li><a href='index.html'>About Me</a></li>
-            <li><a href='projects.html'>Projects</a></li>
-            <li><a href='hobbies.html'>Hobbies</a></li>
-            <li><a class='active' href='exp.html'>Professional Experience</a></li>
-            <li><a href='gallery.html'>Gallery</a></li>
-            <li><a href='blog.html'>Blog</a></li>
+            <li><a href="index.html">About Me</a></li>
+            <li><a href="projects.html">rojects</a></li>
+            <li><a href="hobbies.html">Hobbies</a></li>
+            <li><a class="active" href="exp.html">Professional Experience</a></li>
+            <li><a href="gallery.html">Gallery</a></li>
+            <li><a href="blog.html">Blog</a></li>
         </ul>
     </div>
 
-    <div class='underline'>
-        <h2 class='center'>Ali's Professional Experience</h2>
+    <div class="underline">
+        <h2 class="center">Ali's Professional Experience</h2>
     </div>
-    <div class='wideCenter'>
-        <div class='expInformatic'>
-            <div class='expText'>
+    <div class="wide-center">
+        <div class="experience-box">
+            <div class="experience-text">
                 <p>Software Internship at Google LLC</p>
-                <p style='color: grey'>Summer 2020</p>
+                <p style="color: grey">Summer 2020</p>
```

Comment:
Can you move this style to CSS, and use an id here? (https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns)